sdks/python/apache_beam/io/gcp/bigtableio_read_it_test.py

```diff
@@ -0,0 +1,159 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Is this test working now ? (If so I can try triggering it from the PR).

Reply:
Yeas, you can test it, as long as the extra package name differs from `bigtableio`. Otherwise there is a namespace conflict in which Dataflow will prefer the previous code version that does not yet have the new functionality. See my comments from 24 February.